pkgs/by-name/pr/primitive/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  fetchpatch,
+}:
+
+buildGoModule {
+  pname = "primitive";
+  version = "unstable-2020-05-04";
```

```suggestion
  version = "0-unstable-2020-05-04";
```
see https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning

---

fixed.